src/controllers/graphql/resolvers/user/userResolver.js

```diff
@@ -22,7 +22,12 @@ module.exports = {
         : await authenticatedUserInstance.unFollowSteemUser(args.steemUsernameToUnfollow);
     },
     getProfileInformation: async (_, args) => {
-      return await UserService.getUserInfo(args.name);
+      return await UserService.getUserProfileInfo(args.name);
+    },
+    claimRewardBalance: async (_, args, { authenticatedUserInstance }) => {
```

Comment:
Mentioned this before too, but please break this up into an `if/else`

Multi-line ternaries should be avoided.  They were created so simple `if/else` statements can be made into a clean, one liner. Using a ternary as a multiline completely goes against the purpose it was originally created for. If you have to make a ternary into a multiline because it's too long or complicated etc, it should be refactored into an `if/else` statement.

Using a ternary like this severely limits our ability to customize the `Error` object
ie: `error.code`, `error.description'` or `error.number` etc.

Also, `Error` objects should be thrown, not returned.

If it's an `if/else`, we can do:
```
if (!authenticatedUserInstance) {
   const authError = new AuthenticationError('Error message');
   authError.code = 500;
   throw authError;
}
return await authenticatedUserInstance.claimUsersRewardBalance();
```

We can't be as verbose using a ternary. Let's switch this to an `if/else`, and anywhere else we are returning the `error` rather than throwing it. I remember making a PR doing this refactor, but I don't know if it was merged or not.

I understand it might be annoying, but please take note and remember the comments I leave in PRs so we aren't doing the same redundant review cycle. I'm not talking out of my ass just to sound smart and give you a hard time about coding style. Last thing I want to do is slow down productivity by asking you to refactor code with no reason other than "I don't like how it looks". I'm recommending best practices so that we can write code that can be maintained by other real-world programmers. It's not just to make code look pretty - it has very practical applications